frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -406,8 +411,15 @@ export const TreeViewer = ({ data, heading }) => {
    * @param {Node} clickedNode: The node that was clicked on
    * @param {bool} stayOpen: Whether to keep the infoviewer open if it was
    * already open and the user right-clicked on the same node again.
+   * @param {bool} closeButtonClicked: Whether the current call is coming
+   * from a close button being clicked.
    */
-  const rightClickHandler = (event, clickedNode, stayOpen = false) => {
+  const rightClickHandler = (
+    event,
+    clickedNode,
+    stayOpen = false,
+    closeButtonClicked = false
```

Comment:
Same issue here as with my previous comment about possibly merging `stayOpen` and `closeButtonClicked`

Reply:
^ above comment response ples

Reply:
:heavy_check_mark: